### ISSUE TITLE
Outdated content

### ISSUE THREAD

**Issue by joelcollinsdc** [cheeaun/hackerweb#50]
Hello,

Starting a few weeks ago, I noticed I am seeing stale content when using this web app.  For instance, the top story today is the telegram getting 1M whatsapp users and it says "2 hours ago" even though the story is a few days old.   I've tried pulling down, hitting the refresh button, and reloading the page in chrome.   

If it helps...
Chrome 
iPhone 5
Using wireless data, not wifi

**Comment by cheeaun** [comment]
@joelcollinsdc is it still happening now? May I know the version of the Chrome browser?

**Comment by joelcollinsdc** [comment]
Yes it still is.  I haven't done much troubleshooting yet, but interestingly, the articles updated from what they were before, but are still a few days old.  The second article I see is now the article about not being able to log into Yosemite with an emoji in the password.  

Chrome version is 47.0.2526.70

**Comment by joelcollinsdc** [comment]
Killing chrome and reopening still shows the outdated content.  Safari on my phone does show the latest content...

**Comment by joelcollinsdc** [comment]
FWIW, this morning it was showing updated content.  Well... it says the top story is 16 minutes old on hackerweb and my desktop chrome says its 24 minutes old, but that seems like a non issue.

**Comment by joelcollinsdc** [comment]
It's been working all day.  Maybe it's fixed now.

**Comment by cheeaun** [comment]
@joelcollinsdc glad to know it's fixed for you. I didn't apply any fixes though, as it's kind of hard for me to replicate this bug. I suspect a few issues that might cause this but not 100% sure so I'll keep this in mind for now. Feel free to report here if you're experiencing this bug again, thanks!

**Comment by joelcollinsdc** [comment]
Thanks

**Comment by joelcollinsdc** [comment]
This keeps happening to me.   How can I help troubleshoot?

**Comment by cheeaun** [comment]
@joelcollinsdc Would be great if you could inspect the API calls and try loading them with cache bust e.g.: `http://node-hnapi.herokuapp.com/item/9900168` (could be different endpoint for your case because the app "load-balance" with different endpoints) and try `http://node-hnapi.herokuapp.com/item/9900168?test=1` to see if there are any differences.

**Comment by joelcollinsdc** [comment]
So by loading hackerweb on my desktop and inspecting, i saw that /news and /news2 was being requested.  When I load these from my mobile chrome, I notice updated content.  

Judging from how fast the hackerweb home page loads when the content is stale (very fast), my guess is that something in the JS on the webpage is telling chrome that it doesn't need to refresh the content from local storage.  I was trying to find a way to debug/inspect the local storage in chrome mobile but couldn't...

**Comment by joelcollinsdc** [comment]
Proof i'm not totally crazy.  It always seems to happen in the morning, if that helps.  After the page has been open for an hour or so it seems to fix itself.  (Example, the below pic was taken at 8:30 Central, by 9:30 it was working)

![img_5785](https://cloud.githubusercontent.com/assets/124740/12065736/2c3c3862-afa2-11e5-9ff8-79cb355f1be8.PNG)
